payments_service/auth/tests/test_utils.py

```diff
@@ -0,0 +1,37 @@
+from django.core.exceptions import ObjectDoesNotExist
+
+from payments_service.base.tests import TestCase
+
+from .. utils import set_up_braintree_customer
```

Comment:
relative imports are somewhat new in python so not everyone remembers to use them. I think they are really powerful though because it lets you rename and re-organize modules a lot easier.